stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,45 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+// Main Configuration options for the Susy Grid
+$gutter-width: 20px;
+$column-count: 12;
+
+$susy: (
+  columns: $column-count,
+  use-custom: (rem:true),
+  global-box-sizing: border-box,
```

Comment:
Is `global-box-sizing: border-box` needed if we set `border-box` in `_base.scss`?